python/cudf/cudf/_lib/variant.pxd

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from libcpp cimport bool
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string
+from libcpp.vector cimport vector
+
+
+cdef extern from "<variant>" namespace "std" nogil:
+    cdef cppclass variant:
+        variant& operator=(variant&)
+        size_t index()
+
+    cdef cppclass monostate:
+        pass
+
+    cdef T& get[T](...)
+    cdef T* get_if[T](...)
```

Comment:
Should we alias these as `std_get[_if]`? This function name is so generic without the namespace, especially in Python.

Reply:
I aliased this in the imported code. `orc.pyx` now has `from cudf._lib.variant cimport get_if as std_get_if`.